providers/shared/inputsources/shared/masterdata.ftl

```diff
@@ -984,6 +984,7 @@
             "Rules" : [ "All" ]
           }
         },
+        "OSProfiles": {},
```

Comment:
Does this need to be aligned with our name change?

Reply:
Thanks for the catch - thought I'd already adjusted that. 4f4a8d6 corrects this.